Code/Enigma Final - Complete Version/EnigmaFinal.py

```diff
@@ -324,8 +330,9 @@ def plugboard(mssg,lstplugboard1=dfltPB1,lstplugboard2=dfltPB2):
         #set everything to initial position so it works again for next character in for loop in rotor 3
         reverse=False
 
-def code():
-    global finalmsg, countf, countm, exportCounter, exportRotors
+def code(*event):
```

Comment:
You used `entry.bind("<Return>", code)`,
so you need to specify a optional parameter